src/views/preview/preview.jsx

```diff
@@ -422,13 +459,15 @@ Preview.propTypes = {
         visible: PropTypes.bool
     }),
     canAddToStudio: PropTypes.bool,
-    canCreateNew: PropTypes.bool,
+    canCreateNew: PropTypes.bool, // NOTE: rename these?
```

Comment:
This comment looks like cruft to me... rename what in what way?

Reply:
removed